docs/content/direct/pre-reqs.md

```diff
@@ -53,18 +53,18 @@ follow [these instructions](https://github.com/kubestellar/kubeflex/blob/main/do
 
 ## Additional Software For Running the Examples
 
-- **argocd** - for the examples that use it
+- [**argocd**](https://argo-cd.readthedocs.io/en/stable/getting_started/) - for the examples that use it
 
 ## Additional Software For Building KubeStellar from Source
 
-- **go** version 1.20 or higher - to build Kubestellar
-- **make** - to build Kubestellar and create the Kubestellar image
-- **ko** - to create the Kubestellar image
+- [**go**](https://go.dev/doc/install) version 1.20 or higher - to build Kubestellar
```

Comment:
Let's make the Go version 1.21, because 1.20 is no longer supported. See the policy at https://go.dev/doc/devel/release#policy and the table in https://endoflife.date/go .